telos-frontend/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    jest: true,
+  },
+  parserOptions: {
+    ecmaVersion: 12,
+  },
+  extends: ['plugin:react/recommended', 'airbnb', 'prettier'],
+  plugins: ['react', 'prettier'],
+  rules: {
+    'prettier/prettier': ['warn', { endOfLine: 'auto' }],
+    'react/jsx-filename-extension': ['warn', { extensions: ['.js', '.jsx'] }],
+    'react/react-in-jsx-scope': 'off',
+    'react/jsx-uses-react': 'off',
+    'func-names': 'off',
+  },
```

Comment:
Can you explain what each of these rules does?

Reply:
Was getting errors as App.js doesn't import React.
Apparently it is no longer required to import React in the new versions of React.

https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html
> If you are using eslint-plugin-react, the react/jsx-uses-react and react/react-in-jsx-scope rules are no longer necessary and can be turned off or removed.

`func-names` is off so that you can write anonymous functions without assigning a name to it:
`function() {};` vs `function foo() {}`

Reply:
Sweet, looks good to me